qemu/tests/blk_commit.py

```diff
@@ -33,14 +38,18 @@ def start(self):
             self.test.fail("no active job found")
         logging.info("block commit job running, with limited speed {0} B/s".format(default_speed))
 
-    def create_snapshots(self):
+    def create_snapshots(self, create_file=True):
         """
         create live snapshot_chain, snapshots chain define in $snapshot_chain
+
+        :param create_file, create file inside guest or not
         """
         image_format = self.params["snapshot_format"]
         snapshots = self.params["snapshot_chain"].split()
+        guest_file_name = self.params["file_names"].split()[0]
```

Comment:
In your cfg, `file_names` only has one file, and you only need the very first one.
 I think you don't need to split it.

Reply:
@hereischen That's for further multiple blocks case extension, but here in this case yes, it does not need this, I will drop the split for now and add it when need later. Thanks.